src/components/organisms/NewProfileModal/components/ProfileModalChangePassword/ProfileModalChangePassword.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useCallback } from "react";
+import { FieldErrors, useForm } from "react-hook-form";
+
+import { UserProfileModalFormData } from "types/profileModal";
+import { ContainerClassName } from "types/utility";
+
+import { userProfileModalFormProp as formProp } from "utils/propName";
+
+import { ProfileModalInput } from "../ProfileModalInput";
+import { ProfileModalSectionHeader } from "../ProfileModalSectionHeader";
+
+import "./ProfileModalChangePassword.scss";
+
+interface Props
+  extends ContainerClassName,
+    Pick<ReturnType<typeof useForm>, "register" | "getValues"> {
+  errors?: Pick<
+    FieldErrors<UserProfileModalFormData>,
+    "oldPassword" | "newPassword" | "confirmNewPassword"
+  >;
+}
+
+export const ProfileModalChangePassword: React.FC<Props> = ({
+  containerClassName,
+  register,
+  getValues,
+  errors,
+}: Props) => {
+  const newPasswordValidation = useCallback(
+    (password: string) => {
+      if (getValues(formProp("oldPassword")) && !password)
+        return "New Password cannot by empty";
+
+      if (!password) return;
```

Comment:
return some error here?

Reply:
If the user doesn't want to change the password they can leave all three fields (oldPassword, newPassword, confirmNewPassword) empty and we should not do anything. 
This `if (!password) return` handles this case for oldPassword and newPassword fields.

It can be rewritten like this (logic stays the same):
```
      const oldPassword = getValues(formProp("oldPassword"))
      if (oldPassword && !newPassword)
        return "New Password cannot by empty";

      if (!oldPassword && !newPassword) return;

```

Rewrote in https://github.com/sparkletown/sparkle/pull/1934/commits/fc5d7a6d0ccf9f5d62b06fda37be2c48e95691d0